sensor_msgs/CompressedImage image
---
vision_msgs/Detection2DArray detections